packages/contractkit/src/providers/celo-private-keys-subprovider.ts

```diff
@@ -92,17 +103,17 @@ export class CeloPrivateKeysWalletProvider extends PrivateKeyWalletSubprovider {
     } else {
       debug(`Signer is ${txParams.from} and is one  of ${this.getAccounts()}`)
     }
-    if (txParams.chainId == null) {
+    if (txParams.chainId === undefined || txParams.chainId === null) {
```

Comment:
the common idiom in js is to use `==` to check both null and undefined.
I think is less verbose and evident to any js developer. In fact, it's the only case for `==` our lint rules permit (which are standard)

Reply:
Is that preferable in this case then? Or is it better to be more verbose?

Reply:
i think it's preferable to use `==`

Reply:
Changed to `if (txParams.chainId == null) {`